File: gcc/testsuite/c-c++-common/gomp/requires-1.c
#pragma omp requires unified_address
#pragma omp requires unified_shared_memory
#pragma omp requires unified_shared_memory unified_address
#pragma omp requires dynamic_allocators,reverse_offload

int i;

void
foo ()
{
  if (0)
    #pragma omp requires unified_shared_memory unified_address
    i++;
  #pragma omp requries atomic_default_mem_order(seq_cst)
}

/* { dg-prune-output "not supported yet" } */

File: gcc/testsuite/gfortran.dg/gomp/requires-1.f90
subroutine foo
!$omp requires unified_address
!$omp requires unified_shared_memory
!$omp requires unified_shared_memory unified_address
!$omp requires dynamic_allocators,reverse_offload
end

subroutine bar
!$omp requires unified_shared_memory unified_address
!$omp requires atomic_default_mem_order(seq_cst)
end
